packages/core/src/ctype/CType.ts

```diff
@@ -93,15 +98,17 @@ export function getIdForSchema(
  * @param object Data to be verified against schema.
  * @param schema Schema to verify against.
  * @param messages Optional empty array. If passed, this receives all verification errors.
+ * @param referencedSchemas If schema contains references ($ref) to other schemas, their definitions must be added here in form of an array.
  */
 export function verifyObjectAgainstSchema(
   object: Record<string, any>,
   schema: Record<string, any>,
-  messages?: string[]
+  messages?: string[],
+  referencedSchemas?: Array<Record<string, any>>
```

Comment:
`validator.addSchema` takes a `Schema` parameter, so is this `Record<string, any>` just that?

Reply:
So not planning to change it?

Reply:
just did! seems fine so far, nothing breaking internally